CMakeLists.txt

```diff
@@ -1627,7 +1662,7 @@ if (LWS_WITH_GENERIC_SESSIONS)
 		      plugins/generic-sessions/assets/index.html
 		DESTINATION share/libwebsockets-test-server/generic-sessions
 			COMPONENT examples)
-	install(FILES plugins/generic-sessions/assets/successful-login.html 
+	install(FILES plugins/generic-sessions/assets/successful-login.html
```

Comment:
Sure. Its the atom editor. I had enabled an option to remove trailing whitespaces.